spec/amp-story-player.md

```diff
@@ -0,0 +1,162 @@
+---
+$category@: presentation
+formats:
+  - websites
```

Comment:
We should remove `websites`, right?  This doesn't yet work on AMP pages.

Reply:
Oh I didn't know what this was for. Removed it.